COMPILE-PROTOS.sh

```diff
@@ -20,10 +20,13 @@ if [ ! -d "protos/api-common-protos" ]; then
   mv api-common-protos-input-contract protos/api-common-protos
 fi
 
-go get github.com/googleapis/gapic-generator-go/cmd/protoc-gen-go_gapic
 go get github.com/golang/protobuf/protoc-gen-go
 go get google.golang.org/grpc
 
+d=$(pwd)
```

Comment:
Are you familiar with 'pushd' and 'popd'? Those might be (very slightly) simpler than saving the $(pwd) to a variable.

Reply:
I've updated it to use pushd/popd. Thanks for the suggestion.